roles/edpm_network_config/templates/ci/extracted_crc_nic2_vlans.j2

```diff
@@ -0,0 +1,31 @@
+---
+{% set mtu_list = [ctlplane_mtu] %}
+{% for network in role_networks %}
+{{ mtu_list.append(lookup('vars', networks_lower[network] ~ '_mtu')) }}
+{%- endfor %}
+{% set min_viable_mtu = mtu_list | max %}
+network_config:
+- type: ovs_bridge
+  name: {{ neutron_physical_bridge_name }}
+  mtu: {{ min_viable_mtu }}
+  use_dhcp: false
+  dns_servers: {{ ctlplane_dns_nameservers }}
+  domain: {{ dns_search_domains }}
+  addresses:
+  - ip_netmask: {{ ctlplane_ip }}/{{ ctlplane_subnet_cidr }}
+  routes: {{ ctlplane_host_routes }}
+  members:
+  - type: interface
+    name: nic2
```

Comment:
> order to consume the right interface - eth1, aka nic2.

You don't need a separate template in the role added for CI for that. It can be set with edpm_os_net_config_mappings/net_config_data_lookup ansible var[1].

[1] https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/plugins/modules/edpm_os_net_config_mappings.py#L67-L69

Reply:
hm, overriding the "meaning" of nic1 then?

Reply:
As written bellow - we can't use that for now, because of the type associated to `ansibleVars`... basically, we'd override the whole value, because it's a string, not a proper mapping :(.